androidTEst/androidTEst/Screens/AllFilesScreen.cs

```diff
@@ -0,0 +1,116 @@
+        private readonly By ListOfFiles = By.XPath("//android.widget.TextView[@resource-id='com.nextcloud.client:id/Filename']");
+        private IButton MoreMemu(int index) => ElementFactory.GetButton(By.XPath($"(//android.widget.ImageView[@resource-id='com.nextcloud.client:id/overflow_menu'])[{index}]"), "MoreMemu");
+        private IButton MoreMemuButt = ElementFactory.GetButton(By.XPath("//android.widget.ImageView[@resource-id='com.nextcloud.client:id/overflow_menu']"), "MoreMemuButt");
+        private readonly IButton DelButton = ElementFactory.GetButton(By.XPath("(//android.widget.LinearLayout[@resource-id='android:id/content'])[8]"), "DelButton");
+        private readonly IButton SecondDelButton = ElementFactory.GetButton(By.Id("android:id/button1"), "SecondDelButton");
+        private readonly ITextBox Search = ElementFactory.GetTextBox(By.Id("com.nextcloud.client:id/search_text"), "Search");
+        private readonly ITextBox SearchTextBox = ElementFactory.GetTextBox(By.Id("com.nextcloud.client:id/search_src_text"), "SearchTextBox");
+        private ILabel FileTitle(string fileName) => ElementFactory.GetLabel(By.XPath($"//android.widget.TextView[@resource-id= 'com.nextcloud.client:id/title' and contains(@text, '{fileName}')]"), "FileTitle");
+        private By ListOfFilesWtihFormat(string format) => By.XPath($"//android.widget.TextView[@resource-id= 'com.nextcloud.client:id/title' and contains(@text, '{format}')]");
+
+        public AllFilesScreen() : base(By.Id("com.nextcloud.client:id/fab_main"), "AddNewFileButt")
+        {
+
+        }
+
+        public void GetAllFiles()
+        {
+            MenuButt.Click();
+            AllFilesButt.Click();
+        }
+
+        public void AddNewFile()
+        {
+            AddNewFileButt.State.WaitForClickable();
+            AddNewFileButt.Click();
+        }
+
+        public void CreateTextDocument()
+        {
+            CreateDoc.Click();
+        }
+
+        public void SetFileName(string textFileName)
+        {
+            TextFileName.ClearAndType(textFileName);
+            CreateButton.Click();
+        }
+
+        public string GetFileName()
+        {
+            return TextFileName.Text;
+        }
+
+        public bool WaitForFileDisplayed(string fileName)
+        {
+            return MyFile(fileName).State.WaitForDisplayed();
+        }
+
+        public void OpenFile(string fileName)
+        {
+            MyFile(fileName).Click();
+        }
+
+        public bool IsFileExists(string fileName)
+        {
+            return MyFile(fileName).State.IsDisplayed;
+        }
+
+        public int GetPositionOfMyFile(string fileName)
+        {
+            var files = ElementFactory.FindElements<ILabel>(ListOfFiles);
+            for (int i = 0; i < files.Count; i++)
```

Comment:
стоит использовать хотя бы foreach, а лучше linq
```suggestion
            var file = files.First(fileLabel => fileLabel.Text == MyFile(fileName).Text);
            return files.IndexOf(file) + 1;
```